docs/guides/primitive-input-output.ipynb

```diff
@@ -0,0 +1,476 @@
+{
```

Comment:
<p> </p><p>1.</p><p>" these data will be either expectation values or probability distributions. " -&gt; "these data will either be expectation values and their error bars in the case of the estimator, or samples of circuit outputs in the case of the sampler"</p><p> </p><p>Most importantly, V2 Sampler (in contrast to V1) does not return distributions, it returns samples from distributions.</p><p> </p><p>2.</p><p> list of expectation values -&gt; array of expectation values</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/1621/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>